docs/pages/workflow/glossary-of-terms.md

```diff
@@ -2,18 +2,99 @@
 title: Glossary of terms
 ---
 
+### Android
+
+The mobile operating system sponsored by Google for use with "Android" devices.
+
 ### app.json
 
-**app.json** is a file that exists for every Expo project and it is used to configure your project, for example the name, icon, and splash screen. [Read more in "Configuration with app.json / app.config.js"](configuration.md)
+An [Expo Config](#expo-config) file.
+
+### Apple Capabilities
+
+Cloud services provided by Apple. These services must be enabled for an application in the [Apple Developer Portal](#apple-developer-portal).
+
+### Apple Developer Portal
+
+Apple's [official website](https://developer.apple.com/) for managing application code signing. EAS Credentials automate most of the common reasons a developer might visit this website when developing an app.
+
+### Auto Capability Signing
+
+A feature of EAS build that automatically enables or disables [Apple capabilities](#apple-capabilities) based on the project's entitlements file. [Learn more](/build-reference/ios-capabilities/).
+
+### Autolinking
+
+A cross-platform tool for automatically linking native modules to native apps via native package managers.
+
+- On iOS the tool is used in [CocoaPods](#cocoapods) `ios/Podfile` and invoked during `pod install`
+- On Android the tool is used in the `android/app/build.gradle` and invoked during the [Gradle](#gradle) sync process.
+
+There are two versions of Autolinking: Expo
+
+The default [Prebuild template](#prebuild-template) includes support for [Expo Autolinking](#expo-autolinking), and the [Community Autolinking](#community-autolinking) fork.
```

Comment:
```suggestion
There are two versions of Autolinking: Expo

The default [Prebuild template](#prebuild-template) includes support for [Expo Autolinking](#expo-autolinking), and the [Community Autolinking](#community-autolinking) fork.
```

Is the text here supposed to be "Autolinking: Expo". I'm sorry, but I don't understand it.